### ISSUE TITLE
NTR: CT or MRI: mass lesion(s) in cerebral hemisphere (Lab: radiology)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#8892]
New term request CT or MRI: mass lesion(s) in cerebral hemisphere (segal_090803215323): Brain mass, intracranial mass, intracranial lesion  UMLS: 0742078

**Comment by pnrobinson** [comment]
@MickeySegal

this term is possibly a bit different from the other "mass" terms in the HPO in that it can be taken to imply displacement, compression, or destruction of normal brain tissue. What is the intended context? Probably it is good to debundle the simple "Brain mass" from the "Displacement" phenotypes.

**Comment by MickeySegal** [comment]
We use this for meningioma, CNS lymphoma, glioma, brain abscess, neurocysticercosis and so forth.

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6001083
- label: Cerebral hemisphere mass lesion
- parents: [Abnormal cerebral morphology (HP:0002060)](https://hpo.jax.org/app/browse/term/HP:0002060)
- pmid: 
- def: An abnormal growth or accumulation of tissue in a hemisphere of the cerebrum. Typically a mass is an initial finding that will lead to additional workup. Laryngeal masses can be visualized by multiple methods include computed tomography.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/8892
- synonyms:
